platformLibs/src/platform/windows/opengl32.def

```diff
@@ -0,0 +1,5 @@
+depends = windows posix
+package = platform.opengl32
+headers = wtypes.h minwindef.h wtypes.h GL/gl.h GL/wglext.h
```

Comment:
I think we need some header filter here, so that we don't bind irrelevant declarations. Something similar to `headerFilter = GL/**`.

Reply:
I tired `gl/**` but `GL_` constants were not binded. `gl/** GL_/**` would include gl functions and GL_ constants?